playwright/element-web-test.ts

```diff
@@ -293,3 +293,7 @@ export const expect = baseExpect.extend({
         return { pass: true, message: () => "", name: "toMatchScreenshot" };
     },
 });
+
+test.use({
+    permissions: ["clipboard-read"],
+});
```

Comment:
it seems surprising to me that we do this universally rather than for individual tests.

Reply:
It's coming form the initial PR, but it looks like now this permission requested in the default config (playwrigth.config.ts), so I removed it from here